docs/source/blocks/core/grid.md

```diff
@@ -0,0 +1,156 @@
+
+(grid-block)=
+
+# Grid Block
+
+```{versionadded} Volto 17.0.0-alpha.10
+```
+
+Volto comes with a Grid block, it is a block that contains other blocks.
+The Grid block uses a fixed horizontal layout with a configurable maximum items per block, defaulting to four elements.
+
+The Grid block can potentially contain any block type. However, the used blocks might need some styling work to adapt to the horizontal and small widths in the container. For this reason, only these four are included by default: `slate` (text), `image`, `listing` and `teaser`.
+
+## Settings
+
+The Grid has the following settings:
+
+Headline
+: The headline is a heading that appears above the columns of the Grid Block.
+
+## Block configuration
+
+The Grid block has these configuration settings:
+
+templates
+: When you create a new Grid block, it shows the user a list of choices with the initial contents the block should have.
+By default, it allows the user to choose how many blocks (columns) the block will contain initially.
+One can customize the choices providing a function that generates the initial list of choices and the outcome of choosing one option or another.
+It has the following signature: (type) => (intl) => TemplateData[]
+See the `src/components/manage/Blocks/Grid/template.jsx` for more info.
+```ts
+type TemplateData = {
+    image: Icon,
+    id: string,
+    title: string,
+    blocksData: BlocksData,
+}
+
+type BlocksData = {
+  blocks: {
+    [key: string]: object;
+  };
+  blocks_layout: {
+    items: string[];
+  };
+};
+```
+
+maxLength
+: The maximum elements length allowed inside the grid block.
+By default is four elements.
+
+allowedBlocks
+: Array of block types allowed in the grid block.
+This can be extended to include any of the registered blocks
+
+blocksConfig
+: It allows you to customize the blocks config available for the inner blocks.
+You have to pass the whole existing Volto `blocksConfig` of the main configuration, then modify it given your needs and pass it down.
+You could add different variations, schemaEnhancers, etc or remove them as well but only for blocks inside the grid block.
+```js
+config.blocks.blocksConfig.gridBlock.gridAllowedBlocks: ['teaser', 'image', 'slate'];
+config.blocks.blocksConfig.gridBlock.blocksConfig: {
+  // One could customize the blocks inside the grid like this:
+  ...config.blocks.blocksConfig,
+  teaser: {
+    ...config.blocks.blocksConfig.teaser,
+    variations: [
+      {
+        id: 'default',
+        isDefault: true,
+        title: 'Default',
+        template: DefaultBody,
+      },
+      {
+        id: 'variation2',
+        title: 'variation #2',
+        template: DefaultBody2,
+      },
+    ],
+  },
+};
+```
+
+## Block Image Styling customization
+
+By default, the images inside a Grid block are using a 16/9 aspect ratio.
+It is configurable by setting a LESS variable `@grid-images-aspect-ratio` or setting a CSS property somewhere in your project: `--grid-images-aspect-ratio`.
```

Comment:
And what if I don't want a fixed aspect ratio? Set it to `unset`?

Reply:
Yes, we can document this. But by doing it, then the cards look awful (every image with its own height) if you don't set one. I'd say let's be opinionated on this one, document the override.